cinema/models.py

```diff
@@ -1,10 +1,36 @@
 from django.db import models
 
 
+class Genre(models.Model):
+    name = models.CharField(max_length=63, unique=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Actor(models.Model):
+    first_name = models.CharField(max_length=63)
+    last_name = models.CharField(max_length=63)
+
+    def __str__(self) -> str:
+        return f"{self.first_name} {self.last_name}"
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    actors = models.ManyToManyField(Actor, related_name="movies")
+    genres = models.ManyToManyField(Genre, related_name="movies")
 
     def __str__(self):
         return self.title
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
```

Comment:
would be a good point to throw a check where we won't let creating halls with negative number of anything.